doc/man3/BIO_s_connect.pod

```diff
@@ -4,6 +4,7 @@
 
 BIO_set_conn_address, BIO_get_conn_address,
 BIO_s_connect, BIO_new_connect, BIO_set_conn_hostname, BIO_set_conn_port,
+BIO_set_conn_ip_family, BIO_get_conn_ip_family
```

Comment:
trailing comma missing.

Reply:
Haha. Damn you Oxford commas!
(one learns something new every day :)

Also, the *con_ip_family() functions aren't public according to the doc check, so I just remove them from the documentation.

Reply:
:)  Maybe @levitte and/or @kroeckx have a view on whether those should be public or not?

Reply:
Unless @levitte or @kroeckx step in, this pull request should be ready to merge.

Reply:
Errr, say what?  They are macro definitions present in `include/openssl/bio.h`, so the doc checker is faulty...  or rather, they need to be added in `util/private.num`.  Would you be willing to add them there and add them back in the docs?

Reply:
Sure, I'll do that.

Reply:
I added the BIO_{set,get}_conn_ip_family macros to both the docs and util/private.num.
I did the same for other "missing" macros, such as BIO_get_peer_name.